### ISSUE TITLE
about datasets

### ISSUE THREAD

**Issue by huyanzhuolu** [prismformore/expAT#1]
Sorry to bother you. Could you tell me that how to use it on Market-1501 dataset

**Comment by prismformore** [comment]
@huyanzhuolu  Thank you for your question. If you want to use it on the Market-1501 dataset, I recommend this excellent [repository](https://github.com/michuanhaohao/reid-strong-baseline). And then what you to do is to replace the loss with:
 https://github.com/prismformore/expAT/blob/c132eeb4e67c9d8af8033c426f0df7206a326291/config/criterions.py#L6

**Comment by huyanzhuolu** [comment]
Thank you very much!
But which loss should be replaced?
















在 2020-11-21 17:03:13，"prism" <notifications@github.com> 写道：

@huyanzhuolu Thank you for your question. If you want to use it on the Market-1501 dataset, I recommend this excellent repository. And then what you to do is to replace the loss with:
https://github.com/prismformore/expAT/blob/c132eeb4e67c9d8af8033c426f0df7206a326291/config/criterions.py#L6

**Comment by prismformore** [comment]
@huyanzhuolu  You are welcome. You should replace the triplet loss, which is the counterpart of expAT loss.
Using expAT loss and softmax loss as loss functions work in my experiments. 
BTW, don't forget to do the common space batch normalization on the feature fed to both expAT loss and softmax loss.

**Comment by prismformore** [comment]
@huyanzhuolu I have just updated the code for training and testing on Market-1501 and DukeMTMC-reID datasets. Please take a look!

**Comment by huyanzhuolu** [comment]
Oh Thank you!!!



在 2020-12-12 19:25:09，"prism" <notifications@github.com> 写道：

@huyanzhuolu I have just updated the code for training and testing on Market-1501 and DukeMTMC-reID datasets. Please take a look!

**Comment by prismformore** [comment]
If you have any question, plz let me know. I will close this.
